### PR TITLE
hiclass 4.11.0 :snowflake:

### PR DESCRIPTION
**Pull request by ifitchet**
hiclass 4.11.0 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-5598]
- dev_url:        https://github.com/scikit-learn-contrib/hiclass/tree/v4.11.0
- conda_forge:    https://github.com/conda-forge/hiclass-feedstock/blob/main/recipe/meta.yaml
- pypi:           https://pypi.org/project/hiclass/4.11.0
- pypi inspector: https://inspector.pypi.io/project/hiclass/4.11.0

### Explanation of changes:

- fork conda-forge
- tweaks:
  - `.gitignore` to allow `abs.yaml`
  - update deps
    - push `ray-core` into `run_constrained`
- add upstream tests
  - **WARNING** these include connections to https://zenodo.org (a CERN dataset sharing site) -- not all CI workers can connect

[PKG-5598]: https://anaconda.atlassian.net/browse/PKG-5598?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,30 +10,69 @@ source:
...
+    #- xarray ==2023.1.0  # we have 2023.6.0
+
+# WARNING: these tests include connections to zenodo.org (a CERN
+# dataset sharing site).  Not all CI workers can connect.
+
+{% set tests_to_ignore = "" %}
+# We don't have pyfakefs
+{% set tests_to_ignore = tests_to_ignore + " --ignore=tests/test_LocalClassifiers.py" %}
+# s390x doesn't have shap
+{% set tests_to_ignore = tests_to_ignore + " --ignore=tests/test_Explainer.py" %}
```
Comment (lorepirri) [inline recipe/meta.yaml]: If I get the comment, we want this to be ignored only on `s390x`

```suggestion
{% set tests_to_ignore = tests_to_ignore + " --ignore=tests/test_Explainer.py" %}  # [s390x]
```

**Review by lorepirri — APPROVED** [review]
LGTM,
small nitpick around a test for s390x

**Review by psteyer — APPROVED** [review]
LGTM

File: recipe/meta.yaml (hunk 1 above)
Comment (ifitchet) [reply recipe/meta.yaml]: You're right -- it certainly excluded it from the `s390x`! 😆 

The actual test has a conditional import of `shap` and then toggles usage.  I think one of the tests is missing a skip.

**Pull request merged**